src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS
+ * properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const changes = sourceString;
+
+  changes.split(';')
+    .filter(elem => elem.trim().length !== 0)
+    .map(elem => {
```

Comment:
Bad way to use the map method. The map should use if you need to create `new array` with the results of calling a provided function on every element in the calling array. If you need to do something for each element on the array, but you don’t expect new array, should use forEach